app/src/main/java/com/infomaniak/drive/ui/MainActivity.kt

```diff
@@ -287,6 +287,28 @@ class MainActivity : BaseActivity() {
         LocalBroadcastManager.getInstance(this).unregisterReceiver(downloadReceiver)
     }
 
+    private fun displayInformationPanel() {
+        if (!hasDisplayedInformationPanel) {
+            UISettings(this).apply {
+                val destinationId = when {
+                    !hasDisplayedSyncDialog && !AccountUtils.isEnableAppSync() -> {
+                        hasDisplayedSyncDialog = true
+                        if (AppSettings.migrated) R.id.syncAfterMigrationBottomSheetDialog else R.id.syncConfigureBottomSheetDialog
+                    }
+                    !hasDisplayedCategoriesInformationDialog -> {
+                        hasDisplayedCategoriesInformationDialog = true
+                        R.id.categoriesInformationBottomSheetDialog
+                    }
+                    else -> null
+                }
+                destinationId?.let {
```

Comment:
No need for a variable, you can just do it like this

```suggestion
                }?.let {
```